config/routes.rb

```diff
@@ -0,0 +1,15 @@
+Rails.application.routes.draw do
+  devise_for :users
+  resources :home, only: [:index, :show]
+  resources :users do
+    resources :foods
+    resources :recipes do
+      member do
+        patch 'toggle'
+      end
+      resources :recipe_foods
+    end
+    resources :public_recipes
```

Comment:
- [ ] Please remove the endpoints you don't use as you did in the home path on line 3. This is for best practices.

Reply:
@tugsem 

Done! thanks for the suggestion!